apps/dashboard/src/app/login/onboarding/AccountForm.tsx

```diff
@@ -81,15 +82,21 @@ export const AccountForm: React.FC<AccountFormProps> = ({
           data,
         });
       },
-      onError: (err) => {
-        const error = err as Error;
+      onError: (error) => {
+        console.error(error);
```

Comment:
Remove this

Reply:
That's intentional - I prefer to log errors because toast goes away after some time - but log stays, so its easier to share with support team for a user. It also has the full error object and not just message - so easier to debug